views/partials/nav.gohtml

```diff
@@ -4,47 +4,25 @@
   <label>Rules</label>
   <ul class="nav nav-sidebar">
     {{range .Rulebar}}
-    <li>
-      <a href="{{.Base}}">
-        {{.Name}}
-        {{range .Products}}
-        <span class="badge pull-right {{if eq . " Protect"}} badge-info {{else}} badge-success {{end}}">
-          {{.}}
-        </span>
-        {{end}}
-      </a>
-    </li>
+      {{if and .Sinks (index .Sinks 0).Name }}
+      <li>
+        <a href="{{.Base}}">
+          {{.Name}}
+          {{range .Products}}
+          <span class="badge pull-right {{if eq . " Protect"}} badge-info {{else}} badge-success {{end}}">
+            {{.}}
+          </span>
+          {{end}}
+        </a>
+      </li>
+      {{else}}
```

Comment:
if there are no sinks, show as greyed out italic with a tooltip indicating it's not implemented